contributors/devel/issues.md

```diff
@@ -1 +1,222 @@
+specific release milestone.
+
+For general contributor information on how to triage new incoming
+issues, see the contributor guide section on
+[issue triage](/contributors/guide/issue-triage.md)
+
+The process for shepherding issues into the release by the owner,
+release team, and GitHub bot is outlined below.
+
+## Definitions
+
+- *issue owners*: creator, assignees, and user who moved the issue into a release milestone
+- *Y days*: refers to business days (using the location local to the release-manager M-F)
+- *code slush*: starts when master branch only accepts PRs for release milestone.  No additional feature development is merged after this point.
+- *code freeze*: starts ~1 week after code slush and continues for ~2 weeks.  Only critical bug fixes are accepted into the release codebase.
+
+## Requirements for adding an issue to the milestone
+
+**Note**: Issues with unmet label requirements will automatically be removed from the release milestone.
+
+When adding an issue to a milestone, the Kubernetes [mungegithub
+bot](http://git.k8s.io/test-infra/mungegithub/mungers/milestone-maintainer.go)
+will check that the required labels are set and, if any are found
+missing or inconsistent, comment on the issue with instructions on
+reconciling the labelling problem.  The bot will attempt to contact
+the issue creator regularly and repeatedly via GitHub before
+automatically removing the issue from the milestone.
+
+Label categories:
+
+- SIG label owner
+- Priority
+- Issue type
+
+### SIG owner label
+
+The SIG owner label defines the SIG to which the bot will escalate if the issue is not resolved
+or updated by the deadline.  If there are no updates after escalation, the
+issue may automatically removed from the milestone.
+
+e.g. `sig/node`, `sig/multicluster`, `sig/apps`, `sig/network`
+
+**Note:**
+  - For test-infrastructure issues use `sig/testing`.
+  - For GKE and GCE issues use `sig/gcp` once it is created, and `sig/cluster-lifecycle` until then.
+
+### Priority
+
+Priority label used by the bot to determine escalation path before moving an issues
+out of the release milestone.  Also used to determine whether or not a release should be
+blocked on the resolution of the issue.
+
+- `priority/critical-urgent`: Never automatically move out of a release milestone; continually escalate to contributor and SIG through all available channels.
+  - considered a release blocking issue
+  - code slush: issue owner update frequency: every 3 days
+  - code freeze: issue owner update frequency: daily
+  - would require a patch release if left undiscovered until after the minor release.
+- `priority/important-soon`: Escalate to the issue owners and SIG owner; move out of milestone after several unsuccessful escalation attempts.
+  - not considered a release blocking issue
+  - would not require a patch release
+  - will automatically be moved out of the release milestone at code freeze
```

Comment:
it appears the bot removes these out of the milestone after 4 days - should that be mentioned here?

Reply:
added